### ISSUE TITLE
The Symbol CORE:RMDIR Has No Function Bound to It

### ISSUE THREAD

**Issue by Shinmera** [drmeister/clasp#155]
We're not quite there yet with Quicklisp on d13d2308ca2ac24ec49a3d0e58f21efa96f91b6c.
```
> (ql:update-dist "quicklisp")

Condition of type: SIMPLE-PROGRAM-ERROR
The symbol CORE:RMDIR has no function bound to it

Available restarts:
(use :r1 to invoke restart 1)

1. (RESTART-TOPLEVEL) Go back to Top-Level REPL.

Broken at frame[163] TOP-LEVEL.
 File: #P"/home/linus/.cache/shirakumo/autobuild/clasp-testing/.autobuild/d13d2308ca2ac24ec49a3d0e58f21efa96f91b6c/build/clasp/Contents/Resources/lisp/kernel/lsp/top.lsp" (Position #21029)
```

**Comment by drmeister** [comment]
Added core:rmdir along with core:chmod in 68f27cc

**Comment by Shinmera** [comment]
Fixed on 68f27ccd801d475b4d77aef177905d74cc11ac19
